charts/index.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+entries:
+  trow:
+  - apiVersion: v2
+    appVersion: 0.3.0
+    created: "2020-06-10T09:49:07.248818+02:00"
+    description: A Helm chart for Trow Registry
+    digest: e3b5edcffc082dbdc0b42d5353fd46e7f6984dee1542615bfe351f41eb82178a
+    name: trow
+    type: application
+    urls:
+    - https://github.com/ContainerSolutions/trow/releases/download/v0.3.0/trow-0.3.0.tgz
+    version: 0.3.0
+  - apiVersion: v2
+    appVersion: 0.2.0
+    created: "2020-06-10T09:49:07.248114+02:00"
+    description: A Helm chart for Trow Registry
+    digest: 4674f2dc7a6ce700b27b322d16df31c6db38fdf4dcaacba685892483a34b57bc
+    name: trow
+    type: application
+    urls:
+    - https://github.com/ContainerSolutions/trow/releases/download/v0.2.0/trow-0.1.1.tgz
```

Comment:
Groan, looks like I made a mess of the versions in the 0.2 release. But the chart doesn't actually use these files, right